src/optimize.jl

```diff
@@ -2,6 +2,7 @@ const compiled_calls = Dict{Any,Any}()
 
 # Pre-frame-construction lookup
 function lookup_stmt(stmts::Vector{Any}, @nospecialize arg)
+    # this convert a statement into something else, without the slightly interest in correctness:
```

Comment:
😆 
```suggestion
    # this converts a statement into something else, without the slightest interest in correctness:
```
I remember this one being horrible. It's just enough to squeak by tests.

Reply:
yes, the comment is mostly as a warning against someone using it for new purposes